cmd/snap-confine/ns-support.c

```diff
@@ -769,3 +850,23 @@ void sc_wait_for_helper(struct sc_mount_ns *group)
 	sc_message_capture_helper(group, HELPER_CMD_EXIT);
 	sc_wait_for_capture_helper(group);
 }
+
+void sc_store_ns_info(const sc_invocation * inv)
+{
+	FILE *stream SC_CLEANUP(sc_cleanup_file) = NULL;
+	char info_path[PATH_MAX] = { 0 };
+	sc_must_snprintf(info_path, sizeof info_path,
+			 "/run/snapd/ns/snap.%s.info", inv->snap_instance);
+	stream = fopen(info_path, "wt");
```

Comment:
If you are removing 't', here is another one.

Reply:
And *then* we port to win32 ;-)